### PR TITLE
pkp/pkp-lib#830 fix author dashboard

### PR DESCRIPTION
**Pull request by NateWr**
This PR builds on top of your author dashboard fix branch, @asmecher, and just adds my formatting/style fixes.

There were no changes required in `omp` so you can merge your fix in that repo whenever you'd like.

### CODE REVIEW

File: controllers/tab/authorDashboard/AuthorDashboardTabHandler.inc.php [hunk 1]
```diff
@@ -68,23 +68,24 @@ function fetchTab($args, $request) {
...
-		// Get the last review round.
-		$lastReviewRound = $reviewRoundDao->getLastReviewRoundBySubmissionId($submission->getId(), $currentStage);
-
-		// Create and assign add file link action.
-		if ($fileStage && is_a($lastReviewRound, 'ReviewRound')) {
-			import('lib.pkp.controllers.api.file.linkAction.AddFileLinkAction');
-			$templateMgr->assign('uploadFileAction', new AddFileLinkAction(
-				$request, $submission->getId(), $currentStage,
-				array(ROLE_ID_AUTHOR), null, $fileStage, null, null, $lastReviewRound->getId()));
+		$templateMgr->assign('lastReviewRoundNumber', $this->_getLastReviewRoundNumber($submission));
```
Comment (beghelli) [inline controllers/tab/authorDashboard/AuthorDashboardTabHandler.inc.php]: getLastReviewRoundNumber call is missing the stage id parameter

File: templates/workflow/workflow.tpl [hunk 2]
```diff
@@ -16,22 +15,21 @@
...
 
-<div id="submissionWorkflow">
+<div id="submissionWorkflow" class="pkp_submission_workflow">
 
-{url|assign:submissionHeaderUrl op="submissionHeader" submissionId=$submission->getId() stageId=$stageId contextId="submission" escape=false}
-{load_url_in_div id="submissionHeaderDiv" url=$submissionHeaderUrl class="pkp_page_header"}
+	{url|assign:submissionHeaderUrl op="submissionHeader" submissionId=$submission->getId() stageId=$stageId contextId="submission" escape=false}
+	{load_url_in_div id="submissionHeaderDiv" url=$submissionHeaderUrl class="pkp_page_header"}
 
-{include file="controllers/notification/inPlaceNotification.tpl" notificationId="workflowNotification" requestOptions=$workflowNotificationRequestOptions}
```
Comment (beghelli) [inline templates/workflow/workflow.tpl]: why removing this? Is this a decision to not use in place notifications anymore?

**Comment by NateWr** [comment]
Thanks @beghelli, I've fixed those two issues now.

File: templates/workflow/workflow.tpl (hunk 2 above)
Comment (asmecher) [inline templates/workflow/workflow.tpl]: This is just a change in indentation.

**Comment by asmecher** [comment]
Thanks, @NateWr and @beghelli!

File: templates/workflow/workflow.tpl (hunk 2 above)
Comment (beghelli) [inline templates/workflow/workflow.tpl]: sure, after he got it back here https://github.com/NateWr/pkp-lib/commit/1663c35df13e833b05e8c954e07b6f593f5a6c29 :)

Comment (asmecher) [inline templates/workflow/workflow.tpl]: Ah, usually github nukes comments on outdated commits.

**Pull request merged**